pages/login.tsx

```diff
@@ -3,6 +3,7 @@
 // import { useState } from "react";
 import Menu from "../components/menu";
 import Wallet from "./wallet";
+// import LOGIN from "../components/login-btn";
 
 export default function Login() {
   // const { wallet, connected } = useWallet();
```

Comment:
why this is commented?

Reply:
sir this is commented because the user is checked on the mint page whether he is logged in or not if yes then he proceeds if not he is required to sign in.